src/pudl/analysis/record_linkage/eia_ferc1_model_config.py

```diff
@@ -5,10 +5,9 @@
 model.
 """
 
-import splink.duckdb.comparison_level_library as cll
-import splink.duckdb.comparison_library as cl
-import splink.duckdb.comparison_template_library as ctl
-from splink.duckdb.blocking_rule_library import block_on
+import splink.comparison_level_library as cll
+import splink.comparison_library as cl
+from splink import block_on
```

Comment:
There was a big namespace reorganization in v4. Same modules and functions, but imported from different places.